proto/slip0044/enum.proto

```diff
@@ -0,0 +1,346 @@
+syntax = "proto3";
+package slip0044;
+option go_package = "github.com/trisacrypto/trisa/pkg/slip0044";
+
+// Note: This is not a complete list of all SLIP-0044 registered coin types, but should
+// contain all coins supported by TRISA and TRP. See
+// https://github.com/satoshilabs/slips/blob/master/slip-0044.md for the full list.
+enum CoinType {
```

Comment:
I ended up writing a quick and dirty Python script to parse out the coin names and the coin types. It turned out to be a bit annoying because there were duplicate symbols... so it's not a complete enumeration but we should have the commonly used ones represented here.

Reply:
Looks good; thanks for writing that script!